core/condition.go

```diff
@@ -50,6 +50,9 @@ const (
 
 	// DomainExpirationPlaceholder is a placeholder for the duration before the domain expires, in milliseconds.
 	DomainExpirationPlaceholder = "[DOMAIN_EXPIRATION]"
+
+	// ExitCodePlaceholder is a placeholder for the exit code of a command.
+	ExitCodePlaceholder = "[EXIT_CODE]"
```

Comment:
It's kind of hacky, but what do you think about exposing the exit code through the status instead? The fact that it's technically not an exit code since there is no binary used for ssh doesn't really sit all that well with me.

I imagine most people that check for ssh are going to use `[CONNECTED] == true` anyways.

Reply:
It looks like your PR summary mentions you were going to use `[STATUS]`, but the code uses `[EXIT_CODE]` instead.

Reply:
> It's kind of hacky, but what do you think about exposing the exit code through the status instead? The fact that it's technically not an exit code since there is no binary used for ssh doesn't really sit all that well with me.
> 
> I imagine most people that check for ssh are going to use `[CONNECTED] == true` anyways.

The second commit removes the exit code and only uses `[STATUS]` as you requested, but I forget to remove this reference, sorry.

I'm updating the PR, and we will squash all commits together.